editor/src/core/model/element-metadata-utils.spec.tsx

```diff
@@ -711,9 +730,70 @@ export const App = (props) => {
       path,
       { [EP.toString(path)]: element },
       {},
+      {},
     )
     expect(actualResult).toEqual(true)
   })
+  it('returns false for a component with property controls info including supportsChildren false', () => {
```

Comment:
Can you add a test for both the true case and the default (i.e. unspecified) cases please

Reply:
I added a test using the supportsChildren=true annotation. Regarding the default case: all the existing tests in the file already test that case thoroughly.